### ISSUE TITLE
What does this plugin do?

### ISSUE THREAD

**Issue by 0x7FFFFFFFFFFFFFFF** [salcosta/servicenow-sync#8]
What does this plugin do? The `README` file doesn't tell much. Could  you improve it? I'm interested in using ServiceNow in Sublime. Thanks.

**Comment by dalestubblefield** [comment]
I agree, but for now the best information is here -> http://snaug.com/?p=127
